user_templates/mcu_defaults/main/Fysetc_Spider_v1.x.cfg

```diff
@@ -44,4 +44,3 @@ aliases:
     LIGHT_OUTPUT=MCU_E1_OUT         ,
     LIGHT_NEOPIXEL=MCU_RGB_B        ,
     STATUS_NEOPIXEL=MCU_5V_RGB_DATA ,
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates from the install script